src/Raven.Server/Web/System/AdminServerWideHandler.cs

```diff
@@ -54,14 +54,22 @@ public async Task PutServerWideBackupConfigurationCommand()
             using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
             {
                 var configurationBlittable = await context.ReadForMemoryAsync(RequestBodyStream(), "server-wide-backup-configuration");
-                var configuration = JsonDeserializationCluster.ServerWideBackupConfiguration(configurationBlittable);
+                var observedConfiguration = JsonDeserializationCluster.ServerWideBackupConfiguration(configurationBlittable);
 
-                ServerStore.LicenseManager.AssertCanAddPeriodicBackup(configuration);
-                BackupConfigurationHelper.UpdateLocalPathIfNeeded(configuration, ServerStore);
-                BackupConfigurationHelper.AssertBackupConfiguration(configuration);
-                BackupConfigurationHelper.AssertDestinationAndRegionAreAllowed(configuration, ServerStore);
+                ServerStore.LicenseManager.AssertCanAddPeriodicBackup(observedConfiguration);
 
-                var (newIndex, _) = await ServerStore.PutServerWideBackupConfigurationAsync(configuration, GetRaftRequestIdFromQuery());
+                var existingTasks = GetTaskConfigurations(OngoingTaskType.Backup, JsonDeserializationCluster.ServerWideBackupConfiguration);
+
+                if (existingTasks
```

Comment:
Better move this logic to the `ClusterStateMachine` when accepting a new backup task

Reply:
Done